src/ui/public/notify/notifier.js

```diff
@@ -363,12 +394,50 @@ Notifier.prototype.banner = function (msg, cb) {
   return this.set({
     type: 'banner',
     title: 'Attention',
-    markdown: formatMsg(msg, this.from),
+    content: formatMsg(msg, this.from),
     lifetime: Notifier.config.bannerLifetime,
     actions: ['accept']
   }, cb);
 };
 
+/**
+ * Display a custom message
+ * @param  {String} msg
+ * @param  {Object} config
+ * @param  {Function} cb
+ *
+ * config = {
+ *   title: 'Some Title here',
+ *   type: 'info',
+ *   actions: [{
+ *     text: 'next',
+ *     callback: function() { next(); }
+ *   }, {
+ *     text: 'prev',
+ *     callback: function() { prev(); }
+ *   }]
+ * }
+ */
+Notifier.prototype.custom = function (msg, config, cb) {
+  const customActionMax = 3;
```

Comment:
Since the other "helper" methods take 2 parameters, this one should probably throw if `config` here isn't an object, so that nobody tries to pass a callback as the second parameter. For the custom notification, not passing a config object doesn't make a lot of sense...